content/docs/command-reference/fetch.md

```diff
@@ -17,23 +17,23 @@ positional arguments:
 
 ## Description
 
-`dvc fetch` downloads DVC-tracked files from remote storage into the cache of
-the project (without placing them in the <abbr>workspace</abbr>). This makes
-them available for linking (or copying) into the workspace (refer to
-[dvc config cache.type](/doc/command-reference/config#cache)).
+Downloads DVC-tracked files from remote storage into the cache of the project
+(without placing them in the <abbr>workspace</abbr>, like `dvc pull` would).
+This makes them available for linking (or copying) into the workspace (refer to
+[`dvc config cache.type`](/doc/command-reference/config#cache)).
 
-Without arguments, this ensures that all the files needed for all `.dvc` files
-and stages (in `dvc.yaml` and `dvc.lock`) in the workspace exist in the cache
-(the `--all-branches` and `--all-tags` enable using multiple workspace
-versions).
+Without arguments, `dvc fetch` ensures that the files specified in all
+`dvc.lock` and `.dvc` files in the workspace exist in the cache. The
+`--all-branches`, `--all-tags`, and `--all-commits` options enable fetching data
+for multiple versions, not just the current branch.
```

Comment:
current branch - current workspace

Reply:
OK. I didn't want to use that phrase at it seems redundant (workspace = the "current" checked out stuff). Simplified to "multiple workspace versions" now.